tests/GeneralStateTestsRunner.js

```diff
@@ -54,6 +77,9 @@ function runTestCase (testData, t, cb) {
       }
     },
     function (done) {
+      if (options.jsontrace) {
+        console.log('{ "steps": [' + steps + ']}')
```

Comment:
I  think you'd need to use `JSON.stringify(steps)` here too? Perhaps:
```
JSON.stringify({ "steps": steps })
```

I am also wondering which is faster / uses less memory (not sure which we are optimising for), keeping the array elements stringified or as a map?

Reply:
Agreed.  That is preferable.  I do seem to remember discovering test cases which produce enough steps to crash ethereumjs-vm when trying to print so many steps in one statement.

To use less memory, it might be good to loop through each element in `steps` and print individually.

However, I think this suggestion is good for now until I can identify which tests causes the VM to crash with tracing on.